pkg/runs/runsGet_test.go

```diff
@@ -1107,6 +1209,10 @@ func TestRunsGetURLQueryWithInvalidResultSuppliedReturnsError(t *testing.T) {
 	requestor := ""
 	result := "garbage"
 
+	mockUsername := "tester"
+	mockEnv := utils.NewMockEnv()
```

Comment:
you could have added a parameter to the NewMockEnv call also... but it's not important, might have made it cleaner. Up to you.